src/components/Settings/index.tsx

```diff
@@ -221,11 +221,13 @@ export function Settings({ simple }: { simple?: boolean }) {
                     toggleExpertMode()
                     setShowConfirmation(false)
                   }}
-                >
-                  <TYPE.body fontSize="13px" fontWeight={600} color="text1" id="confirm-expert-mode">
-                    Turn on Expert mode
-                  </TYPE.body>
-                </ButtonError>
+                  content={
+                    <TYPE.body fontSize="13px" fontWeight={600} color="text1" id="confirm-expert-mode">
+                      Turn on Expert mode
+                    </TYPE.body>
+                  }
+                />
```

Comment:
You mean just renaming it or reverting to how it was that the content between brackets displays? 
- Because if you mean latter then you can't pass text as props and test the button. And its very widely used convention to do it like this.

Reply:
I think it's reverting and passing `children`, instead of using the `prop` content. It's easier to read, I think.